content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md

```diff
@@ -78,7 +78,7 @@ kind: ClusterConfiguration
 kubernetesVersion: v1.16.0
 scheduler:
   extraArgs:
-    address: 0.0.0.0
+    bind-address: 0.0.0.0
```

Comment:
these are essentially just examples and not concrete uses cases - e.g. "setting the address for the secure port of the scheduler"

but, given the insecure port might get removed at some point the change is fine.

/lgtm
/sig cluster-lifecycle

Reply:
> these are essentially just examples and not concrete uses cases - e.g. "setting the address for the secure port of the scheduler"
> 
> but, given the insecure port might get removed at some point the change is fine.
> 
> /lgtm
> /sig cluster-lifecycle

I think such an example will be misleading for beginners, such as me